docs/source/basics/local_development.md

```diff
@@ -1,16 +1,8 @@
 # Development and debugging
 
-When developing a workflow, its useful to be able to easily run your task functions so that you can debug issues more
-effectively and overall, iterate faster. However, since tasks and workflows need to run in a highly custom, specified
-environment (namely, the one defined in your Dockerfile(s)), it isn't always feasible to simply run your workflow
-locally.
+When developing a workflow, its useful to run your task functions before running the entire workflow in the cloud so that you can debug environment and logical issues. However, since tasks run in a different environment than your local computer (namely, the one defined in your Dockerfile(s)), it isn't always feasible or instructive to run your workflows locally.
```

Comment:
```suggestion
When developing a workflow, it's useful to run each task function before running the entire workflow in the cloud to debug environment and logical issues. However, since tasks run in the environment defined by the Dockerfile rather than on your computer, it isn't trivial to run them locally.
```
the phrasing still sucks but can't rewrite it any better without a complete redo which would change voice and obviously fall out of line with the rest of the document